### PR TITLE
Updates to README.md

### PR DESCRIPTION
**Pull request by jawnsy**
* Dependency installation commands that can be easily copied/pasted

Signed-off-by: Jonathan Yu <jawnsy@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,12 +168,14 @@ $ sudo cp bin/* /opt/cni/bin/
 ```
 
 ### Running with kubernetes
-You can run the local version of kubernetes using `local-up-cluster.sh`. After starting `ocid` daemon:
+
+You can run a local version of Kubernetes, powered by CRI-O, using `local-up-cluster.sh`. After starting the `ocid` daemon, run:
+
 ```sh
-EXPERIMENTAL_CRI=true CONTAINER_RUNTIME=remote CONTAINER_RUNTIME_ENDPOINT='/var/run/ocid.sock --runtime-request-timeout=15m' ./hack/local-up-cluster.sh
+EXPERIMENTAL_CRI=true CONTAINER_RUNTIME=remote CONTAINER_RUNTIME_ENDPOINT='/var/run/ocid.sock -runtime-request-timeout=15m' ./hack/local-up-cluster.sh
```
Comment (feiskyer) [inline README.md]: `EXPERIMENTAL_CRI` is not needed now.

**Comment by runcom** [comment]
ping @jawnsy could you rebase this one?

**Comment by mrunalp** [comment]
@jawnsy reping :)

File: README.md [hunk 2]
```diff
@@ -39,29 +39,80 @@ It is currently in active development in the Kubernetes community through the [d
...
+
 ```bash
-$ export GOPATH=~/go
-$ mkdir $GOPATH
-$ go get -d github.com/kubernetes-incubator/cri-o
-$ cd $GOPATH/src/github.com/kubernetes-incubator/cri-o
-$ make install.tools
-$ make
-$ sudo make install
+yum install -y \
```
Comment (mrunalp) [inline README.md]: We don't support apparmor on these distros.

**Comment by jawnsy** [comment]
@mrunalp @runcom @feiskyer Rebased & addressed comments, PTAL?

**Comment by mrunalp** [comment]
LGTM

**Comment by runcom** [comment]
LGTM

**Pull request merged**